package.json

```diff
@@ -51,9 +51,9 @@
     "mocha": "^2.3.3",
     "phantomjs-prebuilt": "^2.1.3",
     "pretty-bytes": "^3.0.1",
-    "react": "^0.14.2",
-    "react-addons-test-utils": "^0.14.2",
-    "react-dom": "^0.14.2",
+    "react": "^0.14.2 || ^15.0.0",
```

Comment:
not a good idea to write ` || ` version in devDependency,
It's normal for peerDeps, but devDeps should be more explicit